node/declaration.ts

```diff
@@ -130,8 +130,9 @@ const createDeclarationsFromFunctionConfigs = (
 // Validates and normalizes a pattern so that it's a valid regular expression
 // in Go, which is the engine used by our edge nodes.
 export const parsePattern = (pattern: string) => {
-  // Escaping forward slashes with back slashes.
-  const normalizedPattern = pattern.replace(/\//g, '\\/')
+  // only escape front slashes if they haven't been escaped already
```

Comment:
Tiny nit to make this consistent with the rest of the comments. 🤓 

```suggestion
  // Only escape front slashes if they haven't been escaped already.
```